requirements.txt

```diff
@@ -0,0 +1,3 @@
+Django~=3.2.10
+pylint
```

Comment:
This is not a runtime dependency but a dev/test one. So it shouldn't be shoved into the same file. Make another one and include the direct runtime deps via `-r`.

Reply:
This is addressed. Marking the thread as resolved.